src/cata_utility.h

```diff
@@ -84,6 +84,8 @@ T divide_round_up( units::quantity<T, U> num, units::quantity<T, U> den )
     return divide_round_up( num.value(), den.value() );
 }
 
+static int divide_round_down( int a, int b )
```

Comment:
```suggestion
int divide_round_down( int a, int b )
```

And move the definition of `divide_round_down` from cata_tiles.cpp to cata_utility.cpp and remove the static keyword.